chapters/en/module2.md

```diff
@@ -0,0 +1,789 @@
+
+<choice id="2" >
+
+<opt text="True"  >
+
+What about positions, angles, or shades? There are many channels and it would be difficult to quantify and effectively communicate texture.
+
+</opt>
+
+<opt text="False" correct="true">
+
+Great! We saw that there are several visual channels such as position, angle, area, colour brightness or saturation, and volume!
+
+</opt>
+
+</choice>
+
+</exercise>
+
+
+<exercise id="3" title="To 3D or not to 3D?">
+
+**Question 1**      
+ Which of the following is **NOT** a reason to avoid plotting in 3 dimensions?
+
+<choice id="1" >
+<opt text="Plotting in 3D can distort the data depending on the view.">
+
+This is exactly why 3D plots should be avoided!
+
+</opt>
+
+<opt text="Plotting in 3D can be time consuming." correct="true">
+
+Plotting 3D can be quick to do depending on the tool. 
+
+</opt>
+
+<opt text="3D plots can be more difficult to read than in 2D plots.">
+
+As we saw in the example, a 3rd dimension can reduce the effectiveness of a plot since it's harder to read.
+
+</opt>
+
+
+<opt text="3-dimensions can reduce the clarity of a plot.">
+
+With bar charts, we saw that the observations on the plot appeared less than the true values.
+
+</opt>
+
+</choice>
+
+
+**Question 2**      
+Are there acceptable times to present a 3D plot?
+
+<choice id="2" >
+<opt text="No, never.">
+
+Hang on now ...
+
+</opt>
+
+<opt text="Yes, when the system we are presenting is naturally 3 dimensional.">
+
+What about the blood vessel example we saw?
+
+</opt>
+
+<opt text="Yes, they are usually effective in Topographical Maps."  correct="true">
+
+You got it!
+
+</opt>
+
+
+<opt text="Yes, if we can animated it.">
+
+This can help, but it's not always acceptable.
+
+</opt>
+
+</choice>
+
+
+</exercise>
+
+<exercise id="4" title="Area Charts" type="slides,video">
+<slides source="module2/module2_04" shot="1" start="0:003" end="07:12"> </slides>
+</exercise>
+
+<exercise id="5" title="True or False: Area and lines">
+
+**True or False**       
+*It's always better to use an area plot when showing a single trend over time.*
+
+<choice id="1" >
+
+<opt text="True"  >
+
+Both line and area plots would be fine to visualize a single trend. 
+
+</opt>
+
+<opt text="False" correct="true">
+
+Great! Area plots and line plots are both acceptable for displaying a single trend over time. 
+
+</opt>
+
+</choice>
+
+**True or False**       
+*It's important to be careful to do aggregation when needed or our plots can be misleading.*
+
+
+<choice id="2" >
+
+<opt text="True"  correct="true">
+
+You got it! We saw that when we made a line chart in the slides, it mislead us to believe it was an area chart which we fixed by summing up the y axis column. 
+
+</opt>
+
+
+<opt text="False">
+
+What about when we 
+
+</opt>
+
+
+</choice>
+
+</exercise>
+
+
+<exercise id="6" title='Checking your "Area" of Expertise'>
+
+**Question 1**      
+Before making any plots, what is an important first step in visualizing data?
```

Comment:
Great point about sketching on paper, this is one of the most useful tips I know of myself! Especially in the beginning, it really helps thinking about the data. The first milesone in 532 does this, but it should really be in 531 already.

In the first slide of the course, we have this:

> We often visualize data in order to help us answer a specific question
we have about our dataset,
but it can also help us generate new questions.

I agree that we can have a slide that talks more about this, maybe the first slide before we create some code?I think that would be 1.2, the text could be something like:

> Before creating a visualization, decide why you are making it.
Is there a specific question you are trying to answer,
like comparing the relationship between two dataframe columns?
Or are you creating a plot to help you understand the structure of your data more in general,
such as plotting the distribution of each column?
In either case, it can be *extremely* helpful to draw out your plot with pen and paper first.
This helps you think about if the plot you are creating makes sense
or if there is another plot better suited for the task at hands.
It also makes it easier to write the code afterwards, since you clearly know what you are expecting.

Reply:
Will update question when slide is updated https://github.com/UBC-MDS/exploratory-data-viz/issues/52

Reply:
@hfboyce Did you end up updating this question and or slide? Otherwise I can do it

Reply:
I did not update this question since I wasn't sure what the plan was.